src/app/_components/supported-chains-section.tsx

```diff
@@ -108,7 +114,7 @@ export function SupportedChainsSection(props: { variant: "teal" | "purple" }) {
                 variant={props.variant}
               />
             </div>
-            <Text variant="body" className="capitalize text-light-300">
+            <Text variant="body" className="text-center capitalize text-light-300">
```

Comment:
playing around with the grid i noticed if this text wraps, then it doesn't get centered. Adding this just in case.